src/Traversals/Traversals.jl

```diff
@@ -1,16 +1,16 @@
 module Traversals
 
-# This module does not export much. The primary reasons are that exporting would increase the
-# likelihood of name conflicts, and because the functions here should be used only by developers
-# to create new graph algorithms that rely on breadth-first or depth-first search traversals.
-
 using LightGraphs
+using LightGraphs: getRNG
```

Comment:
If we are about to remove `getRNG` better not use it in this module and pass RNGs directly

Reply:
This PR keeps parity with 1.3.x. The `getRNG` mod should be based on whatever's in the `2.0.0-dev` branch at the time it's merged (likely only this PR, but maybe the next one as well); we can then ensure all future PRs use the correct approach (and can change the existing code in `2.0.0-dev` as necessary).